ec2/sc-ec2-windows-jumpcloud-v1.0.0.yaml

```diff
@@ -125,15 +128,21 @@ Resources:
                 - - 'Powershell.exe Set-ExecutionPolicy Bypass -Scope Process -Force;'
                   - 'Powershell.exe C:\scripts\install-chocolatey.ps1 > C:\scripts\install-chocolatey.log'
             03_install_jq:
-              command: 'Powershell.exe C:\ProgramData\chocolatey\bin\choco install jq --yes'
+              command: 'Powershell.exe C:\ProgramData\chocolatey\bin\choco install jq --yes --no-progress'
             04_install_awscli:
-              command: 'Powershell.exe C:\ProgramData\chocolatey\bin\choco install awscli --yes'
+              command: 'Powershell.exe C:\ProgramData\chocolatey\bin\choco install awscli --yes --no-progress --ignore-checksums'
```

Comment:
Note, awscli install kept failing because checksums didn't match, so had to use `--ignore-checksums`

Reply:
Do you know why the checksums don't match?

Reply:
No, I don't know. We (me and @khai) made some changes yesterday because the install for `awscli` was set up still for 1.*, and the default is now 2.*. After we did that checksums started failing. No idea why. What would you advise?

Reply:
this is a very good explanation why we use the `--ignore-checksums` flag.  https://www.gep13.co.uk/blog/chocolatey-error-hashes-do-not-match